bundles/org.openhab.ui/web/src/components/config/controls/persistence-picker.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <ul class="persistence-picker-container">
+    <f7-list-item :title="title || 'Persistence'" smart-select :smart-select-params="smartSelectParams" ref="smartSelect" v-if="ready">
+      <select :name="name" :multiple="multiple" @change="select" :required="required">
+        <option v-if="!multiple" value="" />
+        <option v-for="service in services" :value="service.id" :key="service.id" :selected="(multiple) ? value.indexOf(service.id) >= 0 : value === service.id">
+          {{ service.label ? service.label + ' (' + service.id + ')' : service.id }}
+        </option>
+      </select>
+    </f7-list-item>
+    <!-- for placeholder purposes before persistence services are loaded -->
+    <f7-list-item link v-show="!ready" :title="title" />
+  </ul>
+</template>
+
+<style lang="stylus">
+.item-picker-container
+  .item-content
+    padding-left calc(var(--f7-list-item-padding-horizontal)/2 + var(--f7-safe-area-left))
+  .item-media
+    padding 0
+  .item-inner:after
+    display none
+</style>
+
+<script>
+export default {
+  props: ['title', 'name', 'value', 'multiple', 'required', 'filterType', 'openOnReady'],
```

Comment:
Indeed, not at the moment. It could be useful when one would allow to configure what persistence service to persist to from the item configuration pages, but that is a long shot. So removing.